iwf/tests/test_basic_workflow.py

```diff
@@ -65,6 +65,11 @@ def get_workflow_states(self) -> StateSchema:
 def test_basic_workflow():
     wf_id = f"{inspect.currentframe().f_code.co_name}-{time.time_ns()}"
 
-    client.start_workflow(BasicWorkflow, wf_id, 100, "input")
+    client.start_workflow(
+        BasicWorkflow,
+        wf_id,
+        100,
+        "input",
```

Comment:
Did you want to add the test here? I feel like just passing the configOverride and no asserting is fine (we did the same Java SDK).

I also think it's better to test AlreadyStartedOptions.
Pass alreadyStartedOptions with a requestId here, and call the same start again, you would expect it return the same runID. 

Then pass a different requestId then you would expect an error thrown

Reply:
I was using this test to do some manual checks. You're right, I'll add it here

Reply:
At the moment, this SDK doesn't return run_id at workflow start, so I can't assert that. Other that this, it's been implemented.